corehq/apps/app_manager/templates/app_manager/partials/settings/app_settings.html

```diff
@@ -274,6 +274,28 @@ <h6>Example:</h6>
           </div>
         {% endif %}
       {% endif %}
+      {% if app.master_is_remote%}
+        <div class="panel panel-appmanager panel-appmanager-form">
+          <form action="{% url "pull_missing_multimedia" domain app.id %}" method="POST">
+            {% csrf_token %}
+            <legend>
+              {% trans "Sync all multimedia with master" %}
```

Comment:
should this have any additional help info to tell users when they need to make use of it?

Reply:
open to suggestions @snopoke. I take that a user would use this if they have missing multimedia and no more updates to be pulled.

Reply:
Related, I don't think this belongs on the main app settings page, because it isn't an action we expect people to be doing often. It could go in the deploy popup - because I think the trigger for using it will be when someone tries to download a CCZ and gets the missing multimedia error from `check_ccz_multimedia_integrity` - or in an area that deals with multimedia.

What if you put it in `multimedia_zip_notice.html`? That's already where we display warnings about missing multimedia. Then it would show up in app settings and also in the deploy popup. I don't think the multimedia tab currently displays for linked apps but I think it'd be reasonable to add it (with just the "Download Multimedia Zip" section, not the "Multimedia Reference Checker" section).

<img width="1098" alt="Screen Shot 2019-08-05 at 10 19 48 AM" src="https://user-images.githubusercontent.com/1486591/62471584-f7018880-b76a-11e9-9ef0-598d39170b8a.png">

Reply:
> because it isn't an action we expect people to be doing often

I think its fine to have none often things under app settings if they are placed in the correct tab, for example "Delete application" is under app settings too.
Actually that probably was the reason subconsciously to put it under "Action" because it has nothing to do with App settings per se but is an action for the app.

I am also fine with making this appear under the `Multimedia Tab` but having it on, each, deploy modal does not seem worth it for the user.
We can show it as a part of the message whenever a CCZ creation fails because it is missing multimedia so that they user knows where to go to resolve if any issue arises.

Reply:
This will also be helpful in the rare case when 
1. the "pull master app request" fails midway in a way that the app doc was updated 
2. there were actually mm changes to be done, but the multimedia pull failed which would result in the case @snopoke is pointing to https://github.com/dimagi/commcare-hq/pull/25010#discussion_r310618033. 
As soon as the user would go to make the CCZ, a message would be presented to sync all mm files.

Reply:
Moved the form under Multimedia tab, https://github.com/dimagi/commcare-hq/pull/25010/commits/30363e0e4d0d403747c48758576b4100dbcbe4d0

Show message to user in https://github.com/dimagi/commcare-hq/pull/25010/commits/b8a76f3e2a19cdf62939a2240e469ccdae4cf955 in case the CCZ fails due to missing multimedia